Features/Stories/Fieldtypes/AuthorsFieldType.feature

```diff
@@ -0,0 +1,180 @@
+        Given a Content Type with an "authors" Field exists
+        When I create a content of this Content Type
+        Then I should have an "name" field
+        And I should have an "email" field
+
+    @javascript
+    Scenario: A Content of a Content Type that has an author fieldtype must be prefilled with user's name and email
+        Given a Content Type with an "authors" Field exists
+        When I create a content of this Content Type
+        Then I should have an "name" field filled with Editor's Username
+        And I should have an "email" field filled with Editor's Email
+
+    @javascript
+    Scenario: A Content of a Content Type that has an author fieldtype must have a "Remove this author" and a "Add another author" buttons
+        Given a Content Type with an "authors" Field exists
+        When I create a content of this Content Type
+        Then I should have an "Remove this author" button
+        And I should have an "Add another author" button
+
+    @javascript
+    Scenario: When editing a Content the label of an author field must have the same name of the field type from the respective Content Type
+        Given a Content Type with an "authors" with "Name" "Participants" exists
+        When I create a content of this Content Type
+        Then I should have a "Participants:" label related with the "eame" and "email" fields
+        And I should have a "Name:" label related with the "name" field
+        And I should have an "Email:" label related with the "email" field
+
+    @javascript
+    Scenario: When editing a Content with an author field the "Remove this author" button is disabled when exists one author
+        Given a Content Type with an "authors" Field exists
+        When I create a content of this Content Type
+        Then all the "Remove this author" buttons are "disabled"
+
+    @javascript
+    Scenario: When editing a Content the "Remove this author" button is enabled when exists more than one author
+        Given a Content Type with an "authors" Field exists
+        When I create a content of this Content Type
+        And I add two authors
+        Then all the "Remove this author" buttons are "enabled"
+
+    @javascript
+    Scenario: When editing a Content with an author field with two authors the "Remove this author" button is disabled again when deleting one author
+        Given a Content Type with an "authors" Field exists
+        And a Content of this Content Type exists with "two" authors
+        When I remove the "First" author
+        Then all the "Remove this author" buttons are "disabled"
+
+    @javascript
+    Scenario: The label of an mandatory author field of a Content must have an * as suffix on the field's label's' name
+        Given a Content Type with a "required" "authors" with "Name" "Participants" exists
+        When I create a content of this Content Type
+        Then I should have a "Participants*:" label related with the "name" and "email" fields

```

Comment:
imho the * is an implementation thing. It's used for ALL fieldtypes, right ?

I'd rather have something like "Then the "Participants" field should be marked as mandatory.

Also, the scenario doesn't mention what page this should appear on.